home_4/src/shipper/strategy/AirEastLetterCostCalculationStrategy.ts

```diff
@@ -0,0 +1,7 @@
+import { BaseCostCalculationStrategy } from "./BaseCostCalculationStrategy";
+
+export class AirEastLetterCostCalculationStategy extends BaseCostCalculationStrategy {
+  protected getCostPerOunce(): number {
+    return 0.39;
```

Comment:
`costPerOunce` is represented as a configuration value rather than a strategy method - so in this particular case it's a bit excessive